src/FormattedText/FormattedText.tsx

```diff
@@ -37,8 +41,8 @@ export default class FormattedText extends React.Component {
     }
   }
 
-  replaceLinksWithA(children) {
-    return flattenDeep([].concat(children).map((child) => {
+  replaceLinksWithA(children: string) {
+    return flattenDeep(([] as string[]).concat(children).map((child) => {
```

Comment:
I'm a bit confused: if `children` is of type `string`, why not just:

```
return this.linkify(child)
```

?

Reply:
it was exactly my doubt...